tests/fingerprints/map.py

```diff
@@ -46,7 +46,9 @@ def test_map_count_fingerprint(smallest_smiles_list, smallest_mols_list):
 def test_map_raw_hashes_fingerprint(smallest_smiles_list, smallest_mols_list):
     map_fp = MAPFingerprint(
         variant="raw_hashes",
-        verbose=0,
+        verbose={
+            "leave": False,
+        },
```

Comment:
You can safely remove `verbose` here entirely

Reply:
Done